src/main/java/frc/robot/systems/BlueMech.java

```diff
@@ -3,39 +3,40 @@
 // WPILib Imports
 
 // Third party Hardware Imports
-import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.Timer;
 
 // Robot Imports
 import frc.robot.TeleopInput;
 import frc.robot.HardwareMap;
 
-public class FSMSystem {
+public class BlueMech {
 	/* ======================== Constants ======================== */
 	// FSM state definitions
 	public enum FSMState {
-		START_STATE,
-		OTHER_STATE
+		IDLE,
+		FIRING
 	}
 
-	private static final float MOTOR_RUN_POWER = 0.1f;
+	private static final int PUSH_TIME = 3;
```

Comment:
include unit of time measurement in variable name

Reply:
Renamed to PUSH_TIME_SECONDS